README.md

```diff
@@ -136,6 +136,11 @@ See [STYLEGUIDE](STYLEGUIDE.md) for file format and coding style guide.
 
 go 1.13+, nodejs/yarn, kubectl
 
+### Backend Development
+The backend is written in go, but its not always clear for developers on how to do integrated debugging with the IDE using the go deleve debugger.
+#### Backend Integrated debugging Guide
+A guide and set shell scripts for integrated IDE debugging of the go backend using VSCode is outlined [here](https://github.com/mitchharpur/openshift-console-debugging).
+
```

Comment:
It would make more sense to copy the documents that you are contributing to this repo probably in contrib/docs.    Maintainers can't maintain them if they reside in an external repo.